custom/opm/reports.py

```diff
@@ -892,14 +891,19 @@ def row_objects(self):
         for index, row in enumerate(self.get_rows(self.datespan), 1):
             try:
                 case_row = self.get_row_data(row, index=1, awc_codes=awc_codes)
-                total_payment += case_row.cash_amt
-                rows.append(case_row)
-            except CaseOutOfRange:
-                try:
-                    rows.append(FakeConditionsMet(row, self, child_index=1, awc_codes=awc_codes))
-                except InvalidRow as e:
-                    if self.debug:
-                        self.add_debug_data(row._id, e)
+                if not case_row.case_is_out_of_range:
```

Comment:
Have you verified that this doesn't change the BPR or CMR?  Not everything is called on `__init__`, so if a property is accessed later which sets the flag to `True`, this check won't catch it.  Is it possible to move this until after all the calculations have happened?